### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by bednarz1990**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,119 @@
...
+            var to = "48987456123";
+            var messageBody = "Have a nice day!";
+
+            // initialize a message API
+            var messageApi = apiFactory.CreateMessageApi();
+
+             // send a text
+             var response = messageApi.SendTextMessage(new PhoneNumber(from), new PhoneNumber(to), messageBody);
+             
+```
```
Comment (barinali) [inline README.md]: Please, take care of the indentations.

File: README.md [hunk 2]
```diff
@@ -1 +1,119 @@
...
+```csharp
+    // initiate ApplicationAction
+    var applicationClient = ApplicationManager.CreateApiFactory(new Credentials(apiKey, apiSecret));
+  
+    //initialize callsApi
+    var callsApi = applicationClient.CreateCallsApi();
+
+     //make a call
+     var guid =  callsApi.CreateCall(new PhoneNumber(callerId), origin, destination, CallType.NUMBER);
+```
```
Comment (barinali) [inline README.md]: Please, take care of the indentations.

File: README.md [hunk 3]
```diff
@@ -1 +1,119 @@
...
+  
+  // get ApplicationAction by id 
+  var appActionById = applicationsApi.GetApplicationActionById(id);
+
+  // get ApplicationAction by api_key assigned to sub-application
+  var appActionByApiKey = applicationsApi.GetApplicationActionByApiKey("apiKey");
+
+  // get ApplicationAction by sub-application name 
+  var appActionByName = applicationsApi.GetApplicationActionByName("testName");
+```
```
Comment (barinali) [inline README.md]: Please, take care of the indentations.

File: README.md [hunk 4]
```diff
@@ -1 +1,119 @@
...
+
+Where `API_KEY` and `API_SECRET` should be replaced with the real key and secret from your Apidaze application.
+
+### Initiate an API action
+To execute any action such as making a call or send text message you need to call method from IApiActionFactory interface. 
+
+```csharp
+  // initialize a message API
+  var messageApi = apiFactory.CreateMessageApi();
+```
```
Comment (barinali) [inline README.md]: Please, take care of the indentations.

File: README.md [hunk 5]
```diff
@@ -1 +1,119 @@
...
+
+## SDK client
+
+### Initiate the SDK api action factory
+
+```csharp
+  // initialize API action factory
+  var credentials = new Credentials(API_KEY, API_SECRET);
+  var apiFactory = ApplicationManager.CreateApiFactory(credentials);
+```
```
Comment (barinali) [inline README.md]: Please, take care of the indentations.

File: README.md [hunk 6]
```diff
@@ -1 +1,119 @@
...
+Scripts builders are used to build XML instructions described in [XML Scripting Reference](https://apidocs.voipinnovations.com).
+To build an instruction which echo back received audio to the caller with some delay use the following code.
+```csharp
+            ApidazeScript script = ApidazeScript.Build().AddNode(new Answer())
+                .AddNode(new Echo { Delay = 500 });
+            var noFormatting = true;
+
+            // Act
+            string xml = script.ToXml();
+```
```
Comment (barinali) [inline README.md]: Please, take care of the indentations.

**Review by barinali — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1 +1,118 @@
...
+
+# Apidaze .NET SDK
+
+The Apidaze .NET SDK contains the C# client of Apidaze REST API as well as an XML script builder.
+The SDK allows you to leverage all Apidaze platform features such as making calls, sending text messages, serving IVR systems and many others in your C# based application.
+The SDK also includes sample applications that demonstrate how to use the SDK interfaces.
+See [Apidaze REST API specification](https://apidocs.voipinnovations.com) which includes XML Scripting Reference as well.
+
+# Requirements
+- .NET Core 3.1
```
Comment (shadzik) [inline README.md:10]: This is really the only requirement?

**Review by shadzik — APPROVED** [review]

**Review by barinali — CHANGES_REQUESTED** [review]
I still think the code blocks should not have any indentations for the first level.

It should be as the following
```csharp
var credentials = new Credentials(API_KEY, API_SECRET);
```
instead of 
```csharp
    var credentials = new Credentials(API_KEY, API_SECRET);
```

File: README.md (hunk 7 above)
Comment (bednarz1990) [reply README.md:10]: Only one to start using SDK.

File: README.md (hunk 2 above)
Comment (bednarz1990) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (bednarz1990) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (bednarz1990) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (bednarz1990) [reply README.md]: Fixed.

File: README.md (hunk 5 above)
Comment (bednarz1990) [reply README.md]: Fixed.

File: README.md (hunk 6 above)
Comment (bednarz1990) [reply README.md]: Fixed.

**Review by barinali — APPROVED** [review]

**Pull request merged**